lua_examples/lfs/_init.lua

```diff
@@ -96,6 +96,4 @@ G.dofile = function(n)
   local mod, ext = n:match("(.*)%.(l[uc]a?)");
   local fn, ba   = index(mod)
```

Comment:
Here (and above), `index` will not refer to `node.flashindex()` because that `local` definition is now scoped within the `_VERSION` check.  I suspect you can rewrite these to use the `LFS` table that's now guaranteed to exist (**utterly untested, almost surely wrong**)...

```
local mod, ext = n:match("(.*)%.(l[uc]a?)");
if ext ~= 'lc' and ext ~= 'lua' then return df(n) else return LFS[mod] or df(n) end
```

Reply:
:rage: that was not a good optimization...